schemas/document-schema.json

```diff
@@ -6,22 +6,19 @@
   "type": "object",
   "definitions":{
     "keyword":{
-      "type":"array",
```

Comment:
Note that this is a defintion of keyword. And later on, when the definition is invoked, it is invoked as an array. So without this change we get an array of arrays of key words which we don't want. All we want is an array of key words